versions.json

```diff
@@ -173,15 +175,16 @@
         "fpm_directories": ["include","lib"]
     },
     "elasticlient" : {
-        "commitish": "3adb172a26baae1a995e810e49fee1688ea44df5",
-        "version_string": "0.1.0",
+        "commitish": "d68e30e382b5f2817be8cd901494736b26d4896e",
+        "version_string": "d68e30e3",
         "license": "MIT",
-        "consortium_build_number": "2",
+        "consortium_build_number": "0",
         "externals_root": "opt/irods-externals",
         "build_steps": [
-            "git submodule update --init --recursive",
+            "cd external/cpr; git checkout 1.7.2",
+            "cd external/jsoncpp; git checkout 1.9.5",
```

Comment:
checks out latest stable releases for submodules.

Reply:
this requires making the change in our forked version or submitting changes to the upstream repo.

this keeps us from having to do either. i'm not advocating for this, but i can say this solves the issue without having to rely on any hard code changes.